sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 7.2.0-beta.2 (Unreleased)
+## 7.2.0 (2021-06-08)
+
+### New Features
+
+- Enable encoding the body of a message to the 'value' or 'sequence' sections (via AmqpAnnotatedMessage.bodyType). Using this encoding is not required but does allow you to take advantage of native AMQP serialization for supported primitives or sequences.
```

Comment:
Maybe add a link to the AMQP Spec that talks about data types?

Reply:
Great idea, added.